src/nnet3bin/nnet3-egs-augment-image.cc

```diff
@@ -76,7 +78,8 @@ struct ImageAugmentationConfig {
  */
 void ApplyAffineTransform(MatrixBase<BaseFloat> &transform,
                           int32 num_channels,
-                          MatrixBase<BaseFloat> *image) {
+                          MatrixBase<BaseFloat> *image,
+			  fill_mode_type fill_mode) {
```

Comment:
I can see you are using tabs. You need to change it to spaces. Fix other places in your code too.